### ISSUE TITLE
Image output of ER diagram is missing

### ISSUE THREAD

**Issue by pea-sys** [pgadmin-org/pgadmin4#6946]
**Describe the bug**

Image output of ER diagram is missing

**To Reproduce**

* 1.  Restore the following databases
https://www.postgresql.org/ftp/projects/pgFoundry/dbsamples/pagila/pagila/
```sql
createdb -U postgres pagilla
psql -U postgres -f pagila-schema.sql pagilla
psql -U postgres -f pagila-insert-data.sql pagilla
psql -U postgres -f pagila-data.sql pagilla
```

* 2. Run ERD for Database with PgAdmin4
![2](https://github.com/pgadmin-org/pgadmin4/assets/49807271/5ee8678a-ac6a-4670-863e-1f77eb71f46d)

* 3. Download Image
* 4. When I open the image, some lines are missing.
I've tried it several times, and the result is the same each time.   

![Untitled](https://github.com/pea-sys/postgresql-expriments/assets/49807271/de1d4be6-bb43-4a21-afee-21358fc30718)

**Expected behavior**

The image is output exactly as displayed.

**Desktop (please complete the following information):**
 - OS:Windows 11
 - Version: 7.6
 - Mode: Desktop

**Comment by adityatoshniwal** [comment]
Hi @pea-sys,
This is fixed with https://github.com/pgadmin-org/pgadmin4/issues/4995 and available from v7.8.
Please upgrade.

**Comment by pea-sys** [comment]
@adityatoshniwal 
thanks
